stumpy/mstump.py

```diff
@@ -1059,6 +1179,15 @@ def mstump(T, m, include=None, discords=False, normalize=True, p=2.0):
         and the Euclidean distance, respectively. This parameter is ignored when
         `normalize == True`.
 
+    T_subseq_isconstant : numpy.ndarray or callable or list, default None
```

Comment:
This first line should be "T_subseq_isconstant : numpy.ndarray, function, or list, default None"